### PR TITLE
Run some tests against Tomcat 7.x

### PR DESCRIPTION
**Pull request by ryanjbaxter**
Mainly to test Servlet 3.0.1 comp…atibility.  See spring-cloud/spring-cloud-netflix#1147

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -52,6 +52,7 @@
 		<module>zuul-config-discovery</module>
 		<module>zuul-proxy</module>
 		<module>zuul-proxy-eureka</module>
+		<module>tomcat-7.x</module>
```
Comment (spencergibb) [inline pom.xml:55]: `zuul-` prefix?

File: tomcat-7.x/src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    example:
+      path: /goodurl/**
+      url: http://localhost:1234/good
+  set-content-length: true
+
+spring:
+  application:
+    name: zuul-app
+server:
+  port: 1234
```
Comment (spencergibb) [inline tomcat-7.x/src/main/resources/application.yml]: Hardcoded port could be brittle.

Comment (ryanjbaxter) [reply tomcat-7.x/src/main/resources/application.yml]: I don't see a good way of getting the random port before the application starts

File: pom.xml (hunk 1 above)
Comment (ryanjbaxter) [reply pom.xml:55]: I figured if there were other tomact 7 tests we wanted to add in the future we could just have one project for them.  If you think that would not be the case I am fine renaming it.

File: tomcat-7.x/src/main/resources/application.yml (hunk 2 above)
Comment (ryanjbaxter) [reply tomcat-7.x/src/main/resources/application.yml]: This should now be addressed in the latest commit

**Review by spencergibb — APPROVED** [review]

**Pull request merged**